omnibus/Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 # Install omnibus software
 group :omnibus do
   gem 'omnibus', git: 'https://github.com/chef/omnibus'
-  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software'
+  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software', branch: 'shahid/sqitch-issue'
```

Comment:
This can be switched back to master now that omnibus-software is current in chef-server